postgres/init/22-portal-content.sql

```diff
@@ -241,6 +243,8 @@ Automotive suppliers must constantly monitor product performance and resolve qua
 Business Partner Services for Fraud & Data Management	Business Partner Services for Fraud & Data Management	f9cad59d-84b3-4880-a550-4072c26a6b93	en
 Automatic monitoring (Early Warning System) of diagnostic data	Automatic monitoring (Early Warning System) of diagnostic data	8488044e-b8df-403a-9cbd-16dad7e4a08c	en
 Works with multiple use cases	Works with multiple use cases	5cf74ef8-e0b7-4984-a872-474828beb5d8	en
+smart MOM (Manufacturing Operations Management) consolidates all production processes and systems for managing manufacturing processes in order to increase efficiency. The smart MOM solution from German Edge Cloud includes various services and enables a holistic production management. This includes, for example, the management of production processes including specific production rules, bill of materials (BOM), recipient data, resource/asset management and much more.
```

Comment:
that dataset 'smart MOM' is incorrectly split into 2 lines (which would import as two separate rows). In addition the separators in between description_short, ap_id and language_short_name are incorrect (spaces instead of tabs)